OpenSim/Simulation/Manager/Manager.h

```diff
@@ -59,7 +59,14 @@ class OSIMSIMULATION_API Manager
     Model *_model;
 
     /** Integrator. */
-    SimTK::Integrator* _integ;
+    // This is the actual integrator that is used when integrate() is called.
+    // Its memory is managed elsewehere; either by the user or by the
+    // _defaultInteg smart pointer.
+    SimTK::ReferencePtr<SimTK::Integrator> _integ;
+
+    // The integrator that is used when using the model-only constructor.
+    // This is allocated only if necessary.
+    SimTK::ResetOnCopy<std::unique_ptr<SimTK::Integrator>> _defaultInteg;
 
```

Comment:
I went ahead and deleted the copy constructor and copy assignment operator. Let me know what you think.